src/rotate.c

```diff
@@ -0,0 +1,26 @@
+#include "rotate.h"
+
+void rotate_block(Block* b, const int rotation_matrix[2][2]){
+    int rotated_cells[4][2];
+
+    for (int i=0;i<4;i++) {     //row of first matrix
+      for (int j=0;j<2;j++) {   //column of second matrix
+      int sum=0;
```

Comment:
Sum should be declared outside for loops and set to 0 inside. Also add spaces to assignments

```
int sum;

for {
...
   sum = 0;
}
```